versioned_docs/version-V1/guides/03-trade-tokens.md

```diff
@@ -123,7 +123,7 @@ const denominatorB = (outputReserveB - outputAmountB) * 997
 const inputAmountB = numeratorB / denominatorB + 1
 
 // Buy ETH with TokenA
-const outputAmountA = userInputEthValue
+const outputAmountA = inputAmountB
```

Comment:
This one was correct originally. We are buying ETH using 'exactOut', so `userInputEthValue` is the amount of ETH that they want as their output